integration-tests/pom.xml

```diff
@@ -65,6 +65,7 @@
         <module>neo4j</module>
         <module>mongodb-client</module>
         <module>jackson</module>
+        <module>json</module>
```

Comment:
Please call it jsonb. And it needs to be added to the native builds pipeline, otherwise it won't be executed on CI.

Reply:
I am surprised I didn't do this the first time around :)

Reply:
Fixed